extension/src/panel.ts

```diff
@@ -267,25 +270,27 @@ export class HistoryDebuggerPanel {
       }
     }
 
-    const type = await fileType(replayerEntrypoint)
-    if (type === undefined) {
+    try {
+      const stat: vscode.FileStat = await vscode.workspace.fs.stat(vscode.Uri.file(replayerEntrypoint))
+      const { type } = stat
+      if (type === vscode.FileType.Directory) {
+        throw new Error(
+          `Configured temporal.replayerEndpoint (${replayerEntrypoint}) is a folder, please provide a file instead`,
+        )
+      }
+      if (type === vscode.FileType.Unknown) {
+        throw new Error(
+          `Configured temporal.replayerEndpoint (${replayerEntrypoint}) is of unknown type, please provide a file instead`,
+        )
+      }
+    } catch {
```

Comment:
Not found isn't the only possible error here, please figure out proper error messages based on the error code.

Reply:
`FileNotFound` is the only error I'm seeing being thrown in `stat` ⬇️ 

https://github.com/microsoft/vscode/blob/1fae59815fe45065718e9d5ae7eb335c9cfba07e/extensions/git/src/fileSystemProvider.ts#L134

I've updated this function to check for that error code and otherwise just show the error message directly for any other edge cases (it will also show the errors thrown in the try block).